unified-runtime/source/adapters/level_zero/platform.hpp

```diff
@@ -126,20 +126,36 @@ struct ur_platform_handle_t_ : public _ur_platform {
   // Structure with function pointers for External Semaphore Extension.
   struct ZeExternalSemaphoreExtension {
     bool Supported = false;
-    ze_result_t (*zexImportExternalSemaphoreExp)(
+    // If LoaderExtension is true, the L0 loader is aware of the MCL extension.
```

Comment:
copy paste error as comment is on "MCL"?

Reply:
fixed in latest